src/io.jl

```diff
@@ -143,7 +143,7 @@ end
 ## E.g. this is very good for `StaticArrays.MVector`s
 
 function fixedlength(t::Type, cycles=IdDict())
-    if isbitstype(t)
+    if isbitstype(t) || Base.isbitstype(t)
```

Comment:
whoa there's a different function in Base?? is it true in all cases as our function?

Reply:
Lol I shouldn’t make PRs at the end of the day...it should be isbitsunion

Reply:
makes much more sense :laughing: